book/doctrine.rst

```diff
@@ -111,8 +111,8 @@ information. By convention, this information is usually configured in an
     of your project, like inside your Apache configuration, for example. For
     more information, see :doc:`/cookbook/configuration/external_parameters`.
 
-Now that Doctrine knows about your database, you can have it create the database
-for you:
+Now that Doctrine can connect to your database server, the following command
```

Comment:
I would remove "server" here as Doctrine also supports database platforms which aren't run on a server (e.g. SQLite, which is commonly used)